helpers.go

```diff
@@ -252,11 +252,26 @@ type commandFunc func(string, *cli.Context)
 // argument as given in ctx.
 func commandInContainer(f commandFunc, ctx *cli.Context) {
 	containerID := ctx.Args().First()
-
 	if container, err := checkContainerRunning(containerID); err != nil {
-		logAndFatalf("%v.\n", err)
+		if err = checkContainerExists(containerID); err != nil {
+			logAndFatalf("%v.\n", err)
+		}
+		logAndPrintf("Checking stopped container %s ...\n", containerID)
+		err = inspectContainer(f, ctx, containerID)
+		if err != nil {
+			logAndFatalf("%v.\n", err)
+		}
 	} else {
-		f(container.Image, ctx)
+		if ctx.GlobalBool("force") {
+			logAndPrintf("WARNING: Force flag used. Manually installed packages will be analyzed as well.\n")
+			err = inspectContainer(f, ctx, containerID)
+			if err != nil {
+				logAndFatalf("%v.\n", err)
+			}
+		} else {
+			logAndPrintf("WARNING: Only the source image from this container will be inspected. Manually installed packages won't be taken into account.\n")
+			f(container.Image, ctx)
+		}
```

Comment:
This function needs some comments. I think that we can move the `inspectContainer(...)` directly after line 254. Please mention in the comments, under which condition(s) `f commandFunc` should be executed.